x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart.tsx

```diff
@@ -58,12 +58,30 @@ export class EmbeddableChangePointChart extends AbstractEmbeddable<
 
   private node?: HTMLElement;
 
+  deferEmbeddableLoad = true;
+
   constructor(
     private readonly deps: EmbeddableChangePointChartDeps,
     initialInput: EmbeddableChangePointChartInput,
     parent?: IContainer
   ) {
     super(initialInput, { defaultTitle: initialInput.title }, parent);
+
+    this.initOutput().finally(() => this.setInitializationFinished());
+  }
+
+  private async initOutput() {
+    const {
```

Comment:
Does this block also need to be in a try catch?

Reply:
no need, as I used `.finally` in the constructor